ipaclient/install/client.py

```diff
@@ -2084,10 +2085,10 @@ def install_check(options):
         try:
             timeconf.check_timedate_services()
         except timeconf.NTPConflictingService as e:
-            print("WARNING: conflicting time&date synchronization service '{}'"
-                  " will be disabled".format(e.conflicting_service))
-            print("in favor of chronyd")
-            print("")
+            logcm("WARNING: conflicting time&date synchronization service '%s'"
+                  " will be disabled", e.conflicting_service)
```

Comment:
Same here

Reply:
No, Thomas is using the recommended form here. See https://docs.python.org/3/library/logging.html#logging.debug